### ISSUE TITLE
Installation - Gardena does not appear to be in HACS 

### ISSUE THREAD

**Issue by stain3565** [py-smart-gardena/hass-gardena-smart-system#36]
I could not find Gardena Smart System in HACS so tried the manual method.

All seemed to be OK until I then I got to the point where I looked in Integrations in Home Assistant and could not find it there either.

Is there an obvious mistake I could have made?

Mark Wingfield

**Comment by stain3565** [comment]
Is there a YAML entry I need for my Gardena client details and defining the system?

**Comment by sabbatho** [comment]
Hi, 
No YAML needed to get it to work.
But make sure you restart Home Assistant between installation in HACS or manual, 
Then make the integration configuration with another restart. 
After that you should have the entity as a vacuum.entity_name and everything working.

Not sure about HACS, i don't seem to be able to search for it (but maybe it's because i already have it installed?).
make sure you for now install the 0.2.1b3 version, since the latest does not work for my as of today 200803.

Hope you get it to work

**Comment by docbobo** [comment]
Same or similar problem here. First I tried installing this via HACS - but Gardena is not showing in the integrations after several restarts. The removed the integration and manually installed it following the instructions. Restarted several times, still not showing up... :( 

Any  more suggestions?

I do see this in the logs:
```
2020-08-18 16:42:58 WARNING (MainThread) [homeassistant.loader] You are using a custom integration for gardena_smart_system which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant.
```

So I suspect the integration is being loaded. But even though I did set up  logging, there is no additional hint  the integration got started.
